InstrumentationEngine.sln

```diff
@@ -688,73 +688,73 @@ Global
 		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.Debug|x86.Build.0 = Debug|Any CPU
 		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.Release|Any CPU.Build.0 = Release|Any CPU
-		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.Release|ARM64.ActiveCfg = Release|ARM64
+		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.Release|ARM64.ActiveCfg = Release|Any CPU
 		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.Release|x64.ActiveCfg = Release|Any CPU
 		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.Release|x64.Build.0 = Release|Any CPU
 		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.Release|x86.ActiveCfg = Release|Any CPU
 		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.Release|x86.Build.0 = Release|Any CPU
 		{F356AF58-F432-415D-BE13-44228F265CEF}.CodeAnalysis.Managed|Any CPU.ActiveCfg = Release|Any CPU
 		{F356AF58-F432-415D-BE13-44228F265CEF}.CodeAnalysis.Managed|Any CPU.Build.0 = Release|Any CPU
-		{F356AF58-F432-415D-BE13-44228F265CEF}.CodeAnalysis.Managed|ARM64.ActiveCfg = Debug|ARM64
+		{F356AF58-F432-415D-BE13-44228F265CEF}.CodeAnalysis.Managed|ARM64.ActiveCfg = Debug|x86
 		{F356AF58-F432-415D-BE13-44228F265CEF}.CodeAnalysis.Managed|x64.ActiveCfg = Release|Any CPU
 		{F356AF58-F432-415D-BE13-44228F265CEF}.CodeAnalysis.Managed|x86.ActiveCfg = Release|Any CPU
 		{F356AF58-F432-415D-BE13-44228F265CEF}.CodeAnalysis.Native|Any CPU.ActiveCfg = Release|Any CPU
 		{F356AF58-F432-415D-BE13-44228F265CEF}.CodeAnalysis.Native|Any CPU.Build.0 = Release|Any CPU
-		{F356AF58-F432-415D-BE13-44228F265CEF}.CodeAnalysis.Native|ARM64.ActiveCfg = Debug|ARM64
+		{F356AF58-F432-415D-BE13-44228F265CEF}.CodeAnalysis.Native|ARM64.ActiveCfg = Debug|x86
 		{F356AF58-F432-415D-BE13-44228F265CEF}.CodeAnalysis.Native|x64.ActiveCfg = Release|Any CPU
 		{F356AF58-F432-415D-BE13-44228F265CEF}.CodeAnalysis.Native|x86.ActiveCfg = Release|Any CPU
 		{F356AF58-F432-415D-BE13-44228F265CEF}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{F356AF58-F432-415D-BE13-44228F265CEF}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{F356AF58-F432-415D-BE13-44228F265CEF}.Debug|ARM64.ActiveCfg = Debug|ARM64
+		{F356AF58-F432-415D-BE13-44228F265CEF}.Debug|ARM64.ActiveCfg = Debug|x86
 		{F356AF58-F432-415D-BE13-44228F265CEF}.Debug|x64.ActiveCfg = Debug|Any CPU
 		{F356AF58-F432-415D-BE13-44228F265CEF}.Debug|x64.Build.0 = Debug|Any CPU
 		{F356AF58-F432-415D-BE13-44228F265CEF}.Debug|x86.ActiveCfg = Debug|Any CPU
 		{F356AF58-F432-415D-BE13-44228F265CEF}.Debug|x86.Build.0 = Debug|Any CPU
 		{F356AF58-F432-415D-BE13-44228F265CEF}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{F356AF58-F432-415D-BE13-44228F265CEF}.Release|Any CPU.Build.0 = Release|Any CPU
-		{F356AF58-F432-415D-BE13-44228F265CEF}.Release|ARM64.ActiveCfg = Release|ARM64
+		{F356AF58-F432-415D-BE13-44228F265CEF}.Release|ARM64.ActiveCfg = Release|x86
 		{F356AF58-F432-415D-BE13-44228F265CEF}.Release|x64.ActiveCfg = Release|Any CPU
 		{F356AF58-F432-415D-BE13-44228F265CEF}.Release|x64.Build.0 = Release|Any CPU
```

Comment:
>Build [](http://example.com/codeflow?start=53&length=5)

oops we still need to build this for x64 otherwise the RemoteUnitTestExecutor project fails to find it

Reply:
Error when building x64: C:\repos\CLRInstrumentationEngine\src\Tests\RemoteUnitTestExecutor\TestEngineBase.cs(95,50): error CS0234: Th
e type or namespace name 'Host' does not exist in the namespace 'RemoteUnitTestExecutor' (are you missing an
assembly reference?)